team_production_system/models/availability.py

```diff
@@ -5,16 +5,28 @@
 
 # Allow mentors to set their avaliabiltiy
 class Availability(models.Model):
+
+    STATUS_CHOICES = [
+        ('Open', 'Open'),
+        ('Requested', 'Requested'),
+        ('Confirmed', 'Confirmed'),
+    ]
+
     mentor = models.ForeignKey(
         Mentor, on_delete=models.CASCADE, related_name='mentor_availability')
     start_time = models.DateTimeField()
     end_time = models.DateTimeField()
+    status = models.CharField(
+        max_length=10, choices=STATUS_CHOICES, default='Open')
+    created_at = models.DateTimeField(auto_now_add=True)
+    modified_at = models.DateTimeField(auto_now=True)
 
     class Meta:
         constraints = [
             UniqueConstraint(
                 fields=['mentor', 'start_time'],
-                name='availability_constraint')
+                name='availability_constraint',
+                violation_error_message='Availability already exists.')
```

Comment:
Nit-pick: Closing parentheses should get it's own line (we could probably configure the linter to do this for us)

Reply:
That is a good plan. I am down to use that formatting.